examples/es_test/paramfile.genic

```diff
@@ -0,0 +1,36 @@
+OutputDir = output_reion # Directory for output
+FileBase = IC              # Base-filename of output files
+
+Ngrid = 128 # Size of cubic grid on which to create particles.
+
+BoxSize = 40000   # Periodic box size of simulation
+MaxMemSizePerNode = 16000
+
+Omega0 = 0.2814      # Total matter density  (at z=0)
+OmegaLambda = 0.7186      # Cosmological constant (at z=0)
+OmegaBaryon = 0.0464     # Baryon density        (at z=0)
+ProduceGas = 1         # 1 = Produce gas  0 = no gas, just DM.
+HubbleParam = 0.697      # Hubble paramater (may be used for power spec parameterization)
+
+Redshift = 25        # Starting redshift
+
+Sigma8 = 0.810      # power spectrum normalization
+
+DifferentTransferFunctions = 0
+ScaleDepVelocity = 0
+
+WhichSpectrum = 2         # "1" selects Eisenstein & Hu spectrum,
+                           # "2" selects a tabulated power spectrum in
+                           # the file 'FileWithInputSpectrum'
+                           # otherwise, Efstathiou parametrization is used
+
+# filename of tabulated input in Mpc/h
+FileWithInputSpectrum = ../../powerspectrum-wmap9.txt
```

Comment:
Should be ../ not ../../ I think

Reply:
But could you please make this class_tk_99 and class_pk_99? and set DifferentTransferFunctions = 1?